ID.me WebVerify SDK/IDmeWebVerify.h

```diff
@@ -105,8 +115,13 @@ typedef NS_ENUM(NSUInteger, IDmeWebVerifyConnection)
 - (void)logout;
 
 /**
- Registers a new connection for the user..
+ Registers a new connection for the user.
  */
 - (void)registerConnectionInViewController:(UIViewController * _Nonnull)viewController scope:(NSString * _Nonnull)scope type:(IDmeWebVerifyConnection)type result:(IDmeVerifyWebVerifyConnectionResults _Nonnull)callback;
 
+/**
+ Registers a new ID for the user.
+ */
+- (void)registerIDInViewController:(UIViewController * _Nonnull)viewController scope:(NSString * _Nonnull)scope type:(IDmeWebVerifyAffiliation)type result:(IDmeVerifyWebVerifyConnectionResults _Nonnull)callback;
```

Comment:
maybe we should use `Affiliation` instead of `ID`to keep consistence with the enum, what do you think?